corehq/apps/hqadmin/static/hqadmin/js/nvd3_charts_helper.js

```diff
@@ -83,7 +83,7 @@ function fill_in_spaces(vals, start, end, interval) {
     // if there are any real values left over then tack them on to the end
     // this should not happen
     if (cur_index < _.filter(vals, function(n) {return n.y > 0;}).length) {
-        ret = ret.concat(vals.slice(cur_index));
+        ret.concat(vals.slice(cur_index));
```

Comment:
Yeah, I figured, that's why I merged.  Was just thinking that it'd be weird if someone comes along later and wonders why this isn't working